Bottom: 5cd4b3e6b55d5befc73bbcea1abaea61a86d255a
Top:    9166d250fc5822ccd1feb5ba2e943ea1dccd56cf
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-23 16:44:03 +0100

Remove commented TEXT_IS_VISIBLE implementation

I'm cleaning up code which uses SWAP_IDENT, and this comment scored a hit.


---

diff --git a/src/macro.h b/src/macro.h
index 7cba5ac..19f80b8 100644
--- a/src/macro.h
+++ b/src/macro.h
@@ -151,12 +151,7 @@ extern int mem_any_set (unsigned char *, int);
 
 /* ---------------------------------------------------------------------------
  *  Determines if text is actually visible
-#define TEXT_IS_VISIBLE(b, l, t)					\
-		(((l)->On && !TEST_FLAG(ONSILKFLAG, (t))) ||     	\
-		(TEST_FLAG(ONSILKFLAG, (t)) && (b)->ElementOn   	\
-		&& ((b)->InvisibleObjectsOn ||                  	\
-		(TEST_FLAG(ONSOLDERFLAG, (t)) != 0) == SWAP_IDENT)))
-*/
+ */
 #define TEXT_IS_VISIBLE(b, l, t) \
 	((l)->On)
